open-api/rest-catalog-open-api.yaml

```diff
@@ -1966,6 +2021,17 @@ components:
               "GET /v1/{prefix}/namespaces/{namespace}/tables/{table}",
               "GET /v1/{prefix}/namespaces/{namespace}/views/{view}"
             ]
+        idempotencyKeyLifetime:
+          type: string
+          format: duration
+          description: >
+            Client reuse window for an Idempotency-Key (ISO-8601 duration, e.g., PT30M, PT24H).
+            Interpreted as the maximum time from the first submission using a key to the last retry
+            during which a client may reuse that key. Servers MUST accept replays for at least this
+            duration and MAY include a small grace period to account for delays/clock skew.
```

Comment:
nit: "small" seems unnecessary in this context.

Reply:
Removed. Thanks